docsite/docs/cli/Project Configs/Credentials.md

```diff
@@ -1,3 +1,40 @@
-# Credentials and profile.yml
+# Credentials and profiles.yml
 
-`fal` integrates with `dbt`'s `profile.yml` file to access and read data from the data warehouse. Once you setup credentials in your `profile.yml` file for your existing `dbt` workflows anytime you use `ref` or `source` to create a dataframe `fal` authenticates using the credentials specified in the `profile.yml` file.
+`fal` integrates with `dbt`'s `profiles.yml` file to access and read data from the data warehouse. Once you setup credentials in your `profiles.yml` file for your existing `dbt` workflows anytime you use `ref` or `source` to create a dataframe `fal` authenticates using the credentials specified in the `profiles.yml` file.
```

Comment:
Some minor corrections:

`fal` integrates with `dbt`'s `profiles.yml` file to access and read data from the data warehouse. Once you setup credentials in your `profiles.yml` file for your existing `dbt` project, anytime you use `ref` or `source` to create a dataframe, `fal` authenticates using the credentials specified in the `profiles.yml` file.